google-cloud-spanner/lib/google/cloud/spanner.rb

```diff
@@ -183,5 +178,8 @@ def self.default_credentials scope: nil
           Spanner::Credentials.default(scope: scope)
       end
     end
+
+    # @private
+    Spanner = Cloud::Spanner unless const_defined? :Spanner
```

Comment:
This needs to be one level up (Under `module Google` rather than `module Cloud`)